src/app/test_executive/block_production_test.ml

```diff
@@ -21,10 +21,14 @@ module Make (Engine : Engine_intf) = struct
   let run network log_engine =
     let open Network in
     let open Malleable_error.Let_syntax in
+    let logger = Logger.create () in
     let block_producer = List.nth_exn network.block_producers 0 in
     let%bind () = Log_engine.wait_for_init block_producer log_engine in
     let%map _ =
       Log_engine.wait_for ~blocks:1 ~timeout:(`Slots 30) log_engine
     in
+    [%log info]
+      "block_production_term: cannot remove '.coda-config': Device or \
```

Comment:
This log message is unusual, did it get garbled somehow?

Reply:
yeah i mis-copy pasted 😅